api/src/main/data/bt_language_names.xml

```diff
@@ -383,6 +712,95 @@ QUESTIONS?
 <iso639-1>mg</iso639-1>
 <script>Latn</script>
 <symbol>MALAGASY</symbol>
+<aux_name0>Malagasy (macrolanguage)</aux_name0>
+</language>
+<language>
+<id>143</id>
+<name>Malagasy, Antankarana</name>
+<doc>Antankarana Malagasy</doc>
+<iso639-3>xmr</iso639-3>
```

Comment:
The correct code is `xmv`. Did you generate this list manually? Maybe there are other small mistakes that I haven’t noticed.